README.md

```diff
@@ -92,11 +98,6 @@ create a server called `FOUNDRY` in your PHP Remote Debug, with the IDE key `xde
 
 ![PhpStorm with xdebug](docs/phpstorm-xdebug-config.png)
 
-### Run tests without docker
-
-If for any reason docker is not available on your computer, the target `make test-fast` will run tests with your local
-php version, and sqlite will be used as database. Results may differ from the CI!
-
```

Comment:
I removed the doc about this because the target `make test-fast` does not exist anymore.

I'd say I'll add it back if someone asks for it :innocent: